### DISCUSSION TITLE
hetzner worker scale

### DISCUSSION THREAD

**Discussion by devMls** [celery/celery#9479] [Ideas]
# Checklist

- [X] I have checked the [issues list](https://github.com/celery/celery/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Issue+Type%3A+Feature+Request%22+)
  for similar or identical feature requests.
- [X] I have checked the [pull requests list](https://github.com/celery/celery/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3A%22PR+Type%3A+Feature%22+)
  for existing proposed implementations of this feature.
- [X] I have checked the [commit log](https://github.com/celery/celery/commits/main)
  to find out if the same feature was already implemented in the
  main branch.
- [X] I have included all related issues and possible duplicate issues
  in this issue (If there are none, check this box anyway).

#### Related Issues

- None

#### Possible Duplicates

- None

# Brief Summary
Celery workers should scale in different "vm" of hetzner

# Design
hetzner offer a python api to create and destroy vm

## Architectural Considerations

None

## Diagrams

N/A

## Alternatives

None

**Comment by auvipy** [comment]
closing this issue as a very vague and incomplete one. and opening a discussion section for this.
